apache/superset
# chore: Remove StyledQueryButton.less

### SUMMARY
Removes the StyledQueryButton less file from the StyledQueryButton component.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Follow instructions on https://superset.apache.org/docs/installation/alerts-reports/#schedule-reports
2. Click on Schedule Query in SQL lab
3. Make sure there are no visual changes

### ADDITIONAL INFORMATION
